docker/lwua-ingest/Dockerfile

```diff
@@ -32,7 +34,5 @@ ENV PATH="/lwua-py/.venv/bin:$PATH"
 # ensure we have the build folder from the poetry stage of this image
 COPY --from=poetry /lwua-py /lwua-py
 
-VOLUME /data
-VOLUME /logging
-RUN ln -s /logging /lwua-py/logging
+RUN rm /lwua-py/logging && ln -s /logging /lwua-py/logging
```

Comment:
just realising: this will need a `rm -f` to force / ignore error, and continue with the && 2nd part
```
RUN rm -f /lwua-py/logging && ln -s /logging /lwua-py/logging
```